SmartDeviceLink/private/SDLVoiceCommandUpdateOperation.m

```diff
@@ -57,6 +57,21 @@ - (void)start {
     }
 
     __weak typeof(self) weakSelf = self;
+
+    // Evaluate the pendingVoiceCommands to check if there is two or more voiceCommands with the same string
```

Comment:
This should be done in `SDLVoiceCommandManager` so that the operation isn't even created if it's not needed. Encapsulate it into a class method there.